stanford_soe_r25.module

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Code for the Stanford SOE R25 module.
+ */
+
+/**
+ * Hook into the reservation form for insidesoe.
+ */
+function stanford_soe_r25_form_stanford_r25_reservation_alter(&$form, &$form_state, $form_id) {
+  $form['stanford_r25_booking_tentative']['#weight'] = -11;
```

Comment:
Please add code comment describing the purpose of this alter so the next developer gets a quick overview of what this is doing.